UpSkill_App/UpSkill.Infrastructure/Data/Models/ApplicationUser.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace UpSkill.Infrastructure.Data.Models
+{
+    using Microsoft.AspNetCore.Identity;
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using UpSkill.Infrastructure.Data.Common.Models;
+
+    public abstract class ApplicationUser : IdentityUser, IAuditInfo, IDeletableEntity
+    {
+        // TODO: Add company logo property
+
+        public ApplicationUser()
```

Comment:
I believe we don't need the constructor, since we don't need to explicitly set the User Id. ASP.NET Core Identity takes care to assign the Id to the newly-created IdentityUser upon persistence. 

Classes that have nothing to do with IdentityUser, and have Id of type string, do need a new Guid upon class object instantiation.

Reply:
Yes, I thought the same thing, but I did it because that is how it looked like in Niki Kostov's ASP.NET Core Template. Now I have removed it.